presentation/src/main/java/me/androidbox/presentation/worker/UploadEventWorker.kt

```diff
@@ -0,0 +1,105 @@
+package me.androidbox.presentation.worker
```

Comment:
WorkManager workers are rather data related, especially if they upload something like here

Reply:
@philipplackner so this should belong in my Data Layer module. I did have it there ordinary but moved it to the presentation module because that is where the work would be triggered from.

Reply:
No, the work could as well be triggered from your data layer in the repository. That's where the request is executed, the worker is just a tool to execute it reliably

Reply:
A way to trigger it from Presentation, would be to abstract it in the Domain layer, and have the implementation on the data layer

Kinda like what you did for the Email Validator